uaa/src/test/resources/sso_metadata.xml

```diff
@@ -0,0 +1,21 @@
+<md:EntityDescriptor ID="y0f_8PtWxWnNp3rlzZT4-VGoQbY" cacheDuration="PT1440M" entityID="gefssstg" xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"><ds:Signature xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
```

Comment:
Please revert this change. This change did not fix SAML acceptance tests and SamlLoginAT is excluded for now.  There is a separate story to fix SAML acceptance tests.  We will update metadata then.

Reply:
Removed